must-gather/collection-scripts/gather

```diff
@@ -24,12 +26,14 @@ resources+=(objectbuckets)
 # Add general resources to list if necessary 
 
 # Run the Collection of Resources using must-gather
-for resource in ${resources[@]}; do
+for resource in "${resources[@]}"; do
```

Comment:
Same here, a fix/improvement that is not related to the actual topic of this commit or even PR.

Reply:
done.